test-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/PropertyTransferDslTest.groovy

```diff
@@ -0,0 +1,111 @@
+package com.smartbear.readyapi4j.dsl
+
+import com.smartbear.readyapi.client.model.PropertyTransfer
+import com.smartbear.readyapi.client.model.PropertyTransferTestStep
+import com.smartbear.readyapi.client.model.TestStep
+import com.smartbear.readyapi4j.TestRecipe
+import org.junit.Test
+
+import static TestDsl.recipe
+import static com.smartbear.readyapi4j.teststeps.propertytransfer.PropertyTransferBuilder.fromPreviousResponse
+import static org.junit.Assert.fail
+
+class PropertyTransferDslTest {
+
+    public static final String URI = "/uri_to_get"
+    public static final String FIRST_STEP = 'theGet'
+    public static final String LAST_STEP = 'thePost'
+    public static final String sourcePath = '$.customer.address'
+    public static final String targetPath = '$.customer'
+
+    @Test
+    void buildsPropertyTransfer() throws Exception {
+        TestRecipe recipe = recipe {
+            GET(URI, {
+                name FIRST_STEP
+            })
+            propertyTransfer(fromPreviousResponse(sourcePath).toNextRequest(targetPath))
```

Comment:
Would just be overloading thanks to different parameter types, so it won't collide.